client/src/modules/users/users.js

```diff
@@ -161,7 +161,10 @@ function UsersController($state, $uibModal, Users, Notify, Modal, uiGridConstant
           return user;
         },
       },
+    }).result.then(() => {
+      loadGrid();
```

Comment:
Looks good!  I prefer the grid refresh to trying to edit the data locally, so this is my preferred fix.  Thanks!